targets/targets.json

```diff
@@ -753,7 +753,12 @@
             },
             "stdio_uart_rx": {
                 "help": "default RX STDIO pins is defined in PinNames.h file, but it can be overridden"
-            }
+            },
+            "lpticker_delay_ticks": {
+                "help": "https://os.mbed.com/docs/v5.9/reference/low-power-ticker.html",
+                "value": 1,
```

Comment:
Why is only one cycle needed?

Reply:
Good idea for the link. Just maybe replace "v5.9" with "latest".

Reply:
In some previous patch, I have removed this delay tick, but it seems that not all the tests can pass...
Then I set back value to 1 as the default value.